be/src/exec/base_scanner.cpp

```diff
@@ -284,6 +330,12 @@ void BaseScanner::close() {
     if (!_pre_filter_ctxs.empty()) {
         Expr::close(_pre_filter_ctxs, _state);
     }
+
+    if (_state != nullptr && _state->enable_vectorized_exec()) {
```

Comment:
no need `_state != nullptr`?

Reply:
> no need `_state != nullptr`?
it just use to protect the _state when another UT pass a nullptr.